source/install/ee-install.rst

```diff
@@ -48,10 +48,12 @@ Upgrading Enterprise Edition to a newer version
 Changing a License Key
 ----------------------
 
-To change an existing license key on an Enterprise Edition server to a new license key:
+Make sure that the new license is for a number of users that is greater than or equal to the current total number of users on your system. To find the total number of users, go to the REPORTING section of the System Console and click **Site Statistics**. The total number of users is in the **Total Users** field. The license is rejected if this value is greater than allowed by the key.
 
-1. In the **System Console**, go to **OTHER** > **Edition and License** > **License Key** and click "Remove Enterprise License and Downgrade Server". This clears the license from the server and refreshes the System Console. 
-2. In the **System Console**, go to **OTHER** > **Edition and License** > **License Key** and upload the new license key file.
-   
-The **Edition** and **License** sections on the page should update to show the capabilities of your new license key. 
+**To install a new license key**:
 
+1. In the OTHER section of the System Console, click **Edition and License**. The *Edition and License* page opens.
```

Comment:
I was thinking that too, at first. But in the System Console, OTHER is not clickable; it's just a label. If it was something that you had to click in order to see its items, then **OTHER** would be correct. Does that make sense?